draft-js-mention-plugin/tsconfig.json

```diff
@@ -1,3 +1,3 @@
 {
-  "extends": "../tsconfig.json"
+  "extends": "../tsconfig.json",
```

Comment:
Trailing commas are not supported in valid JSON. Is that ok?

Reply:
Yes, you can have trailing commas and even comments in tsconfig.json files